docs/sql/statements/copy-to.rst

```diff
@@ -72,13 +72,24 @@ Parameters
 .. _sql-copy-to-table_ident:
 
 ``table_ident``
-  The name (optionally schema-qualified) of the table to be exported.
+  The name (optionally schema-qualified) of the table to be exported. E.g.
+
+  ::
+
+    cr> COPY quotes TO DIRECTORY '/tmp/';
+    COPY OK, 3 rows affected ...
```

Comment:
This looks pretty identical to the example further above, and doesn't exactly highlight the `table_ident`.

Is this really useful to have in addition?

Reply:
Wouldn't using a schema-qualified ident like `doc.quotes` improve it? Otherwise I double on @mfussenegger point that this may not add much value..

Reply:
That's fair, it doesn't add anything. Removed, thanks!